tests/run-macros/i13942/Macro_1.scala

```diff
@@ -0,0 +1,21 @@
+import scala.quoted.*
+
+import reflect.Selectable.reflectiveSelectable
+
+inline def testShowErrType: Unit = ${ testShowErrTypeImpl }
+
+def testShowErrTypeImpl(using q: Quotes): Expr[Unit] = {
+  import q.reflect.*
+
+  type Types = {
+    def UnspecifiedErrorType: Any
```

Comment:
Quite hard to test, has `Types` cannot be imported properly there

Reply:
The best approach is to minimize the macro that hit the error type. Without it, we would not be able to properly test for regressions.